app/views/hiring_staff/vacancies/edit_vacancy_sections/_edit_job_summary.html.haml

```diff
@@ -15,5 +15,6 @@
 
   .app-check-your-answers__contents
     %dt.app-check-your-answers__question#about_school
-      %h4.govuk-heading-s= t('jobs.about_school', school: @vacancy.organisation.name)
+      - about_school_label_organisation = more_than_one_location?(@vacancy.organisations) ? 'the schools' : @vacancy.organisation.name
```

Comment:
I'd change the intention of the helper to encapsulate all of this and remove this assignment from the template